CoolestMovieAPI/CoolestMovieAPI/Controllers/MoviesController.cs

```diff
@@ -25,101 +25,294 @@ public class MoviesController : ControllerBase
         public MoviesController(IMovieRepository movieRepository)
         {
             _movieRepository = movieRepository;
-           
         }
        
+
+        [HttpGet]
+        public async Task<ActionResult<IList<Movie>>> GetAll()
+        {
+            try
+            {
+                var results = await _movieRepository.GetAllMovies();
+
+                if (results.Count == 0)
+                {
+                    return NotFound();
+                }
+                else
+                {
+                    return Ok(results);
+                }
```

Comment:
verbose  '{' ' }' single line if statement dosen't require {}. 

Consider 
if(args)
 return x;


or a ternary operator

Reply:
This is true, I just like it because it's consistent and makes it more obvious.
I will however read up on Ternary operators. I've seen them but never understood them fully